### PR TITLE
Principles expansion

### PR DESCRIPTION
**Pull request by addshore**

### CODE REVIEW

File: PRINCIPLES.md [hunk 1]
```diff
@@ -10,25 +10,36 @@ https://www.mediawiki.org/wiki/Core_Platform_Team/Initiative/Core_REST_API_in_Me
...
-  - Example: `entity/item` or `entity/property`
-  
 ## Operations
 
 https://www.mediawiki.org/wiki/Core_Platform_Team/Initiative/Core_REST_API_in_Mediawiki/Design_principles#Operations
 
-Additional clarification:
+### PATCH
+
+Additional to the Mediawiki core principles we will also use `PATCH`.
```
Comment (itamargiv) [inline PRINCIPLES.md:40]: Should we add more about the reasoning behind choosing patch (wikibase deals with large objects with multiple properties that we would like to expose for updating without assigning uniqe identfiers, etc...)?

**Review by itamargiv — COMMENTED** [review]
So far LGTM, thank you!

File: PRINCIPLES.md (hunk 1 above)
Comment (addshore) [reply PRINCIPLES.md:40]: I did think in that direction but if felt a little bit odd writing that in principles.
I think principles should probably remain light weight and perhaps point to the reasoning for decisions.
However right now we are not including said decisions in this repo.

Comment (itamargiv) [reply PRINCIPLES.md:40]: Gotcha! Cool

**Review by itamargiv — APPROVED** [review]

**Pull request merged**